css/style.css

```diff
@@ -1,14 +1,72 @@
 
- /* Use the 'Roboto' font from Google. Already added to your HTML header */
+#pond{
+display: flex;
+align-items:center;
+justify-content:center;
+font-size: 20px;
 
-body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
 }
-
-/* Add your custom styles here */
-/* Remember: be organised, use comments and separate your styles into meaningful chunks */
-/* For example: General styles, Navigation styles, Hero styles, Footer etc. */
-
-/*  TIP: When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex' */
-
+#img{
+  display:flex;
+  justify-content: center;
+  justify-content:space-around;
+}
+#karma{
+  display: flex;
+  justify-content: center;
+ font-size: 270%;
+}
+#paragraph{
+  display:flex;
+  align-content:center;
+  justify-content:space-around;
+  font-size: 200%;
+}
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+h1 {
+ margin-top: 17.5rem;
+ font-size: 3.125rem;
+ color:white;
+}
+h3 {
+  margin-top: 1.25rem;
+  font-size: 1.563;
+  color: white;
+  font-weight: 300;
+}
+a {
+    display: block;
+    padding: 8px;
+    color: gray;
+    text-decoration: none;
+}
+button {
+  background-color: #f15a29;
+  border-color: #f15a29;
+  width: 200px;
+  height: 50px;
+  color: white;
+  font-size: 1.125rem;
+}
+a:active {
+  color: black
+}
+.imageDiv {
+  background-image: url("/home/jo/bisratdoc/karma-clone/img/first-background.jpg");
+  display: flex;
```

Comment:
this image is using your local computer's path, so it doesn't appear for me

Reply:
done